libtransact/Cargo.toml

```diff
@@ -30,12 +30,12 @@ repository = "http://github.com/hyperledger/transact"
 [dependencies]
 bzip2 = { version = "0.3", optional = true }
 cbor-codec = { version = "0.7", optional = true }
-cylinder = "0.2"
+cylinder = { version = "0.2", optional = true }
 glob = { version = "0.3", optional = true }
 hex = "0.3"
 libc = ">=0.2.35"
 lmdb-zero = ">=0.4.1"
-log = { version = "0.4", features = ["std"] }
+log = { version = "0.4", optional = true, features = ["std"] }
```

Comment:
This should also be mentioned in the commit message

Reply:
Fixed